dts/bindings/pinctrl/telink,b91-pinctrl.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Telink B91 Pin controller node.
+    Based on pincfg-node.yaml binding.
+
+compatible: "telink,b91-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node.yaml
+      child-binding:
+        property-allowlist:
+          - bias-disable
+          - bias-pull-down
+          - bias-pull-up
+
+properties:
+    reg:
+      required: true
+
+    label:
+      required: true
+
+    pad-mul-sel:
+      type: int
+      required: true
+      description: |
+          PinMux pad_mul_sel register values. Pins functions depend on this value.
```

Comment:
nit: `Pin functions`. Can description be improved? For example, how do pin function depend on this value?

Reply:
Done